app/views/admin/organizations/edit.html.erb

```diff
@@ -21,7 +21,7 @@ Editing
 <!-- Default box -->
 <div class="box">
 <div class="box-header with-border">
-  <h3 class="box-title">Update record for <%= current_organization.name %></h3>
+  <h3 class="box-title">Update record for <%= @organization.name %> </h3>
```

Comment:
Why not use `current_organization`?

Reply:
both are the same in this case, but there is a pattern in this project, every place is using @organization instead of current_organization, so considering that is a question about standardization, I replaced current_organization by @organization.

Reply:
both are the same in this case, but everywhere in this project is using @organization instead of current_organization, so, considering that is a question about standardization, I replaced current_organization by @organization.